docker/fork-inducer.yml

```diff
@@ -1,25 +1,35 @@
 services:
   worker-ping:
-    image: worker-ping:dev
+    image: worker-ping:${VERSION:-dev}
+    devices:
+      - "/dev/sgx/provision:/dev/sgx/provision"
+      - "/dev/sgx/enclave:/dev/sgx/enclave"
+    volumes:
+      - "/var/run/aesmd:/var/run/aesmd"
     build:
       context: .
       dockerfile: ping.Dockerfile
-    depends_on: [ 'integritee-node', 'integritee-worker-1', 'integritee-worker-2' ]
+    depends_on: [ 'integritee-node-${VERSION}', 'integritee-worker-1-${VERSION}', 'integritee-worker-2-${VERSION}' ]
     networks:
       - integritee-test-network
     #entrypoint: "ping integritee-worker-2 | while read pong; do echo \"$$(date): $$pong\"; done"
```

Comment:
Could you remove this commented out code as well? Currently it is unused and IIRC the healthcheck is implemented differently as well, so we do not need it.

Reply:
done